client.go

```diff
@@ -132,6 +134,13 @@ func WithKeepAlives(keepAlives bool) ClientOption {
 	}
 }
 
+// WithWellKnownSRVLookups enables federation lookups of well-known and SRV records.
+func WithWellKnownSRVLookups(wellKnownSRV bool) ClientOption {
```

Comment:
Doesn't `WithWellKnownSRVLookups` already imply `true`?

Reply:
You could also supply `WithWellKnownSRVLookups(false)` to force them to be disabled, so no.